app/src/main/java/org/mozilla/fenix/components/toolbar/Toolbar.kt

```diff
@@ -109,7 +110,9 @@ class Toolbar(
                 context.getString(R.string.browser_menu_find_in_page),
                 R.color.icons
             ) {
-                // TODO Find in Page
+                FindInPageIntegration.launch?.invoke()
```

Comment:
We're going to want to use a better solution in the long run than static methods for this. I'll accept this for now, but we may want to use a DI framework like Koin.